mission/para_player_init_client.sqf

```diff
@@ -208,6 +208,58 @@ call vn_mf_fnc_display_location_time;
 // Tutorial System
 [] call vn_mf_fnc_tutorial_subsystem_client_init;
 
+
+//Add Master Arm addAction for Boats and Land Vehicles
+if hasInterface then
+{
+    // Remove action to re-add it later
+    private _old_action_id = player getVariable ["vn_mf_masterarm_action_id", -1];
+    if (_old_action_id > -1) then { player removeAction _old_action_id; player setVariable ["vn_mf_masterarm_action_id", -1]; };
+
+    private _action =
+    [
+        localize "STR_VN_MASTER_ARM_OPEN_ACTION",
+        {
+            params ["_target", "_caller", "_actionId", "_arguments"];
+            ["init"] call VN_fnc_masterarm;
+        },
+        [],
+        -100,
+        true,
+        true,
+        "",
+        "local (vehicle _this) && {!(vehicle _this isKindOf 'Air' || vehicle _this isKindOf 'Man') && {speed (vehicle _this) <= 5 && {(driver vehicle _this) isEqualTo _this && {vn_fnc_masterarm_action_objects findif {(vehicle _this distance _x) < 25} > -1}}}}",
```

Comment:
It should be better performance wise to assigne the vehicle to local variable instead of using the `vehicle` command multiple times. Also you can use `toString <CODE>` for better IDE syntax highlighting and completion in the action code.
```suggestion
        toString {
            private _vehicle = vehicle _this;
            local _vehicle && {
                !(_vehicle isKindOf 'Air' || _vehicle isKindOf 'Man') && {
                    speed _vehicle <= 5 && {
                        driver _vehicle == _this && {vn_fnc_masterarm_action_objects findif {(_vehicle distance _x) < 25} > -1}
                    }
                }
            }
        },
```

Reply:
Oh yeah thats much better, thanks.